test/data8/xml1-output.xml

```diff
@@ -0,0 +1 @@
+<article headline='Final season of "Lost" premieres on Tuesday'/>
```

Comment:
I think we should stick to double quotes as delimiters and replace the double quotes inside with `&quot;`

So the expected output would be:

```xml
<article headline="Final season of &quot;Lost&quot; premieres on Tuesday"/>
```

Or we could offer both options using a config option ?

Reply:
Sounds good! I just pushed a commit along these lines.